test/bin/dune

```diff
@@ -1,6 +1,6 @@
 (cram
  (applies_to :whole_subtree)
- (deps %{bin:opam-monorepo} %{bin:gen-minimal-repo}))
+ (deps %{bin:opam-monorepo} %{bin:gen-minimal-repo} (sandbox preserve_file_kind)))
```

Comment:
Heads up that I added this sandbox configuration for tests. Some of the tests use `sed -i` to modify files in place and this fails when the files are symlinked into the sandbox (fortunately!) which was happening on my mac but not on linux.

Reply:
Didn't know about it, good catch! I was confused why that worked so I just tested it and it looks like GNU sed just recreates the "inplace" file as a regular file instead of following the symlink.